src/wallet/interfaces.cpp

```diff
@@ -91,7 +91,7 @@ WalletTxStatus MakeWalletTxStatus(const CWallet& wallet, const CWalletTx& wtx)
     WalletTxStatus result;
     result.block_height =
         wtx.state<TxStateConfirmed>() ? wtx.state<TxStateConfirmed>()->confirmed_block_height :
-        wtx.state<TxStateConflicted>() ? wtx.state<TxStateConflicted>()->conflicting_block_height :
```

Comment:
I think that `TxStateBlockConflicted` and `TxStateMempoolConflicted` are a bit more clear, because with something like `TxStateUnconfirmedConflict` instead of `TxStateMempoolConflicted`, it is unclear whether the conflict is in the mempool, or just hasn't been broadcasted yet. Additionally, I felt that it was more uniform with the other transaction states to be describing the state of the transaction instead of the state of a conflict. Eg. `TxStateBlockConflicted` indicates that the transaction is block-conflicted, and similarly `TxStateConfirmed` indicates that the transaction is confirmed. On the other hand, `TxStateConfirmedConflict` indicates the state of the conflict. 

That being said, if you or others think that `TxStateConfirmedConflict` or something similar is better, I would be open to changing it.

Reply:
I don’t feel strongly about it, please feel free to mark this comment as resolved.